linode/manager
# refactor: [M3-6287] - `Src > Features > Users`

## Description 📝
MUI v5 migration for `Users`

## Major Changes 🔄
- Ran `tss-react` codemod on `src > features > Users`.
- Replaced default export with named export.
- Removed `index.tsx` file.
- Convert Class Components to Functional Components.
- Major refactor for `UserPermissions.tsx` into smaller files/components.
- Introduced `useImmerReducer` hook to replace `lensPath` from `rambda`.

## How to test 🧪
1. Verify the functionality around the `Users` feature works as expected.
2. Look for any UI defects or regressions in the `Users` landing page.
3. Visit `account/users` and validate adding new users and changing the `User Permissions`.
